data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -9,6 +9,8 @@ fi
 ARTIFACTS="/tmp/artifacts-${GATHER_ID}"
 mkdir -p "${ARTIFACTS}"
 
+exec &> >(tee "${ARTIFACTS}/gather.log")
```

Comment:
Is this esoteric enough to warrant a comment explaining what it is doing?

Reply:
Possibly, but the comment is in the commit message where `git blame` will find it and everyone else can skim over it ;).